app/assets/javascripts/discourse/app/models/category.js

```diff
@@ -492,6 +492,15 @@ export default class Category extends RestModel {
     return [...(parentAncestors || []), this];
   }
 
+  @discourseComputed("parentCategory", "parentCategory.predecessors")
+  predecessors(parentCategory, parentPredecessors) {
```

Comment:
What's the difference between `predecessors` and `ancestors` just above in this file (lines 490-493)?

Reply:
The order is reversed and predecessors doesn't include the category.

Reply:
It would be good to add a test for this, if possible. If anything, as documentation for what it is supposed to return and in what order.